### DISCUSSION TITLE
Editing songtext for multiple titles

### DISCUSSION THREAD

**Discussion by DrMartinus** [strawberrymusicplayer/strawberry#1229] [Ideas]
This may seem strange, but I have done that with Clementine quite often. I don't know how many thousand titles I have on my hard drive, but at some point of time (the collection grew over more than 20 years) the player software scraped automatically the lyrics for the titles and stored them in the meta data. I don't know which software did that - Strawberry is my fourth or fifth media player, since I can't use Clementine because of missing libraries, but that scraping was done by a software before that, I guess. 
Anyway: When the lyrics collections which were scraped did not have any lyrics for a particular title, the software just added a line as songtext which is similar to this: "There are no lyrics for this title available on http://...."
Since this message came from two to three different servers in different wording, it took me some time to figure out how to remove those messages from the lyrics section in the meta data of multiple titles. But I figured it out - in Clementine. I could highlight a bunch of titles, check if they contained those messages or actual lyrics, and if all titles had only this message (...no lyrics...), I highlighted them all and inserted a space in the lyrics field and removed it again. Thus, after saving the changes, none of the titles had this message as lyrics any more. I did that with a few thousand titles, but there are still a lot to go.
Now, in Strawberry I just checked and saw that lyrics are separated (in a separate tab in the meta data dialog). I can not highlight multiple files, because lyrics aren't shown at all if I do that. So I would have to go through each single title in order to remove this useles message. Imagine doing this a hundred times, and then multiply that by 50 to 100 - that's what would be ahead of me, if I attempt to do it with Strawberry.
Maybe I missed something, then I apologize and am grateful for a pointer to a strategy how to solve this. But if not, I would very much appreciate if editing the lyrics in multiple titles would be possible again (as it was in Clementine).

And a final sidenote: I am a Strawbs fan, I notice that Strawberry has it's name derived from that band. Nice! :-)

**Comment by trincowski** [comment]
I don't see much point in allowing this in Strawberry, it's a music player after all... and not a ID3 Tagger.
You can do that using a more appropriate software such as Kid3 (Linux) or EasyTag (Windows). No?

**Reply by DrMartinus** [reply]
Well, I like that Strawberry shows the lyrics while playing a song, and working on lyrics I don't want to do in another software - I see the problem within Strawberry and not elsewhere, and hence Strawberry is the ideal place to fix it. If I want only a player, I don't need a fancy interface. I can let a CLI-software play my music. But I want also to see the lyrics, and maybe some info about the artist now and then. For that, softwares like Starwberry are made - in my opinion.
